Update supported distros in Readme

techguy16 opened the pull request:
Ubuntu 22.04 can run PowerShell, so adding it as supported distro.

<!-- Anything that looks like this is a comment and can't be seen after the Pull Request is created. -->

# PR Summary
In the Readme, there was no "Ubuntu 22.04" as a supported distro in the table. Ubuntu 22.04 does support PowerShell.

<!-- Summarize your PR between here and the checklist. -->

## PR Context
Ubuntu 22.04 users probably thought that PowerShell didn't support their distro so I added it as a row in the table.

<!-- Provide a little reasoning as to why this Pull Request helps and why you have opened it. -->

techguy16 committed:
Adding Ubuntu 22.04 as supported distro

Ubuntu 22.04 can run PowerShell, so adding it as supported distro.

techguy16 commented:
Hi can you please review these changes?

TravisEz13 committed:
Apply suggestions from code review

TravisEz13 committed:
Add Ubuntu 16 back for LTS

TravisEz13 committed:
remove deb9

TravisEz13 committed:
Fix Kali

TravisEz13 committed:
Update README.md

TravisEz13 committed:
Merge branch 'master' into patch-1

TravisEz13 committed:
fix merge issue

daxian-dbw commented on README.md:
```diff
@@ -30,10 +30,10 @@ You can download and install a PowerShell package for any of the following platf
 | -------------------------------------------| ------------------------| ------------------------| ----------------------| ------------------------------|
 | [Windows (x64)][corefx-win]                | [.msi][lts-windows-64]  | [.msi][rl-windows-64]   | [.msi][pv-windows-64] | [Instructions][in-windows]    |
 | [Windows (x86)][corefx-win]                | [.msi][lts-windows-86]  | [.msi][rl-windows-86]   | [.msi][pv-windows-86] | [Instructions][in-windows]    |
+| [Ubuntu 22.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu22]     | [.deb][pv-deb]        | [Instructions](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu)   |
```
Please create a link with `in-ubuntu22`.

techguy16 replied:
Added `in-ubuntu22` link

techguy16 committed:
Update README.md

techguy16 committed:
Update README.md

I recommend adding a link somewhere that users of older operating systems can click and download the correct version of PowerShell for their OS.

daxian-dbw commented on README.md:
```diff
@@ -113,6 +113,7 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 [in-ubuntu16]: https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-linux#ubuntu-1604
 [in-ubuntu18]: https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-linux#ubuntu-1804
 [in-ubuntu20]: https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-linux#ubuntu-2004
+[in-ubuntu22]: https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu
```
Remove the `en-us` part and let the page itself decides which locale to use.
```suggestion
[in-ubuntu22]: https://learn.microsoft.com/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu
```

techguy16 committed:
Update README.md

Co-authored-by: Dongbo Wang <dongbow@microsoft.com>

techguy16 commented:
@microsoft-github-policy-service agree

TravisEz13 committed:
Merge branch 'master' into patch-1

TravisEz13 committed:
Update README.md

TravisEz13 committed:
Update README.md

TravisEz13 committed:
Update README.md

techguy16 commented:
@TravisEz13 No worries - need any help, just ask 😁
